recipes/libgit2/pstdint.h

```diff
@@ -0,0 +1,912 @@
+/*  A portable stdint.h
```

Comment:
Should say that's neat! Are many other such packages conda-forge that patch missing stuff. Could be a good idea to put up some info related those somewhere.